morpheus/_lib/cmake/libmorpheus.cmake

```diff
@@ -165,17 +177,74 @@ set_target_properties(morpheus
     CUDA_STANDARD_REQUIRED ON
 )
 
+if(MORPHEUS_PYTHON_INPLACE_BUILD)
+  morpheus_utils_inplace_build_copy(morpheus ${CMAKE_CURRENT_SOURCE_DIR})
+endif()
+
+# ##################################################################################################
+# - install targets --------------------------------------------------------------------------------
+
+# Get the library directory in a cross-platform way
+rapids_cmake_install_lib_dir(lib_dir)
+
+include(CPack)
+include(GNUInstallDirs)
+
 install(
-  TARGETS
-    morpheus
-  EXPORT
-    ${PROJECT_NAME}-exports
-  FILE_SET
+    TARGETS morpheus
+    EXPORT
+    ${PROJECT_NAME}-core-exports
+    LIBRARY
+    DESTINATION ${lib_dir}
+    FILE_SET
     public_headers
-  COMPONENT
-    Wheel
+    COMPONENT Core
```

Comment:
Why was the component switched from `Wheel` to `Core`?

Reply:
I was trying to follow MRC as an example. It was my understanding that the `Wheel` is all the parts of Morpheus that get installed with the python package. 

While `Core` was used for items such as the headers which are installed by cmake with `--target install`